### PR TITLE
[9.0][MIG] medical physician us

### PR DESCRIPTION
**Pull request by kenwmak**
This is dependent on #69 and #86

### CODE REVIEW

File: medical_physician_us/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (lasley) [inline medical_physician_us/__init__.py]: Use the shorter OCA header

File: medical_physician_us/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (lasley) [inline medical_physician_us/__openerp__.py]: Use the shorter OCA header

File: medical_physician_us/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+{
+    'name': 'Medical Physician - US Locale',
+    'version': '9.0.1.1.0',
```
Comment (lasley) [inline medical_physician_us/__openerp__.py]: 9.0.1.0.0

File: medical_physician_us/models/__init__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (lasley) [inline medical_physician_us/models/__init__.py]: Use the shorter OCA header

File: medical_physician_us/models/medical_physician.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (lasley) [inline medical_physician_us/models/medical_physician.py]: Use the shorter OCA header

**Comment by lasley** [comment]
Rebased and updated to incubator code. This is ready for review.

cc @tedsalmon @kellylougheed

File: medical_physician_us/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    'name': 'Medical Physician - US Locale',
+    'version': '9.0.1.0.0',
+    'author': "LasLabs, Odoo Medical Team, Odoo Community Association (OCA)",
+    'category': 'Medical',
+    'depends': [
+        'medical_base_us',
+        'medical_physician',
+    ],
+    "website": "https://laslabs.com",
+    "licence": "AGPL-3",
```
Comment (tedsalmon) [inline medical_physician_us/__openerp__.py]: `s/licence/license`

File: medical_physician_us/tests/medical_physician.yml [hunk 7]
```diff
@@ -0,0 +1,31 @@
+-
```
Comment (tedsalmon) [inline medical_physician_us/tests/medical_physician.yml]: -- Test data? This should be in `data/`. -- These are actual tests, huh... Regardless, it's not in the manifest :/

**Review by tedsalmon — APPROVED** [review]
👍  Approved pending minor changes requested.

File: medical_physician_us/tests/medical_physician.yml (hunk 7 above)
Comment (lasley) [reply medical_physician_us/tests/medical_physician.yml]: YML tests are deprecated, so this will be removed.

**Comment by lasley** [comment]
There are some warnings in Runbot:
```
2017-01-14 21:53:16,704 216 WARNING openerp_test openerp.models: method medical.physician._check_dea_num: @constrains parameter 'country_id' is not writeable
2017-01-14 21:53:16,704 216 WARNING openerp_test openerp.models: method medical.physician._check_npi_num: @constrains parameter 'country_id' is not writeable
2017-01-14 21:53:18,092 216 WARNING openerp_test openerp.models: method medical.physician._check_dea_num: @constrains parameter 'country_id' is not writeable
2017-01-14 21:53:18,092 216 WARNING openerp_test openerp.models: method medical.physician._check_npi_num: @constrains parameter 'country_id' is not writeable
```

**Review by kellylougheed — APPROVED** [review]

**Comment by lasley** [comment]
Closing - we no longer have interest in v9. We will continue these forward in v10.

Anyone is welcome to pick the code up and reintroduce a PR, however.

**Pull request closed without merging**